.github/workflows/ci.yml

```
@@ -0,0 +1,36 @@
+# .github/workflows/ci.yml
+
+name: Run Tests on Pull Request
+
+on: 
```

yamllint

[error] trailing spaces

(trailing-spaces)